apollo-router/src/plugins/limits/layer.rs

```diff
@@ -19,24 +19,36 @@ pub(super) enum BodyLimitError {
 }
 
 struct BodyLimitControlInner {
-    limit: AtomicUsize,
+    limit: Arc<AtomicUsize>,
     current: AtomicUsize,
 }
 
 /// This structure allows the body limit to be updated dynamically.
 /// It also allows the error message to be updated
+///
```

Comment:
@BrynCooke TODO. Look at BodyLimit in more detail.

Reply:
Fixed in [f5d3bdb](https://github.com/apollographql/router/pull/6486/commits/f5d3bdb4bd96fbdc18ab16a9c00c44b0345a1428)